tests/test_dns.py

```diff
@@ -104,66 +104,58 @@ def test_getaddrinfo_7(self):
         self._test_getaddrinfo(None, 0, type=socket.SOCK_STREAM)
 
     def test_getaddrinfo_8(self):
-        self._test_getaddrinfo('', 0)
-        self._test_getaddrinfo('', 0, type=socket.SOCK_STREAM)
-
-    def test_getaddrinfo_9(self):
-        self._test_getaddrinfo(b'', 0)
-        self._test_getaddrinfo(b'', 0, type=socket.SOCK_STREAM)
```

Comment:
We shouldn't just drop the tests that started to fail with libuv 1.48, as they ensures uvloop behaves identically to CPython asyncio.